macos/FluentUITestApp/TestAvatarViewController.swift

```diff
@@ -4,7 +4,7 @@
 //
 
 import AppKit
-import OfficeUIFabric
+import FluentUI
```

Comment:
we've probably already decided this 8 times (sorry), but we're ok with the macOS and iOS modules having the same name? All code consuming them *should* only be searching in the place for the appropriate platform, but likely most code consuming this doesn't have their search paths as strictly managed as Office does

Reply:
most of the platform that using cocoapod to consume this library via pod file would be something like 
target 'FooiOS' do
    platform :ios, '11.0'
 ...
end

target 'FoomacOS' do
    platform :osx, '10.15'
    ...
end

which should generate the right xcconfigt file for those search path correctly ...

Reply:
Carthage should also make this very simple (and does right now with both frameworks being named "OfficeUIFabric" by detecting the platform and putting it in an appropriate folder in the build output, either `Carthage/Build/iOS` or `Carthage/Build/macOS`. So outside of those two supported package managers supporting it, I'd say we can trust clients rolling their own dependency management to handle this case.